site/src/pages/extrinsic.js

```diff
@@ -101,9 +101,9 @@ function Extrinsic() {
         data={[
           { name: "Extrinsics", path: "/extrinsics" },
           {
-            name:
-              `${extrinsic?.indexer?.blockHeight}-${extrinsic?.indexer?.extrinsicIndex}` ??
-              "...",
+            name: extrinsic
+              ? `${extrinsic?.indexer?.blockHeight}-${extrinsic?.indexer?.extrinsicIndex}`
```

Comment:
maybe we can improve this even better, you see , if `id` is a block-index format, we can use `id` directly, if its a hash, then we will wait for the `extrinsic` from server